jcl/src/java.management/share/classes/java/lang/management/ThreadInfo.java

```diff
@@ -121,119 +89,55 @@ private ThreadInfo(long threadIdVal, String threadNameVal,
 			long lockOwnerIdVal, String lockOwnerNameVal,
```

Comment:
What calls this constructor?

Reply:
Is this constructor still used?

Reply:
This constructor is used by the `from()` factory method.